tests/console/velociraptor_client_event_collect.pm

```diff
@@ -0,0 +1,125 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: velociraptor-client
+# Summary: install velociraptor-client and collect events on server
+# Maintainer: Gayane Osipyan <gosipyan@suse.com>
+
+use base "consoletest";
+use testapi;
+use strict;
+use warnings;
+use version_utils;
+use version_utils qw(is_sle is_tumbleweed);
+use utils;
+use Utils::Architectures;
+use repo_tools "generate_version";
+use serial_terminal "select_serial_terminal";
+use python_version_utils;
+
+sub run {
+
+    select_serial_terminal;
+
+    # get python version
+    my $system_python_version = get_system_python_version();
+
+    # get os version
+    my ($version, $sp, $host_distri) = get_os_release;
+    my $sp_version = "$version.$sp";
+
+    # setup repositories
+    zypper_call "ar -p 90 -f --no-gpgcheck http://download.suse.de/ibs/SUSE:/SLE-15:/Update/standard/ sle15update";
+    zypper_call "ar -p 90 -f --no-gpgcheck http://download.suse.de/ibs/SUSE:/SLE-15-SP4:/Update/standard/ sle15SP4update";
+    zypper_call "ar -p 90 -f --no-gpgcheck http://download.suse.de/ibs/SUSE:/SLE-15-SP1:/Update/standard/ sle15SP1update";
+    if (is_sle) {
```

Comment:
Just is_sle, then you can drop the following `is_tumbleweed` or this condition is pointless.

Reply:
I think it would make more sense to drop this condition and keep TW condition.
Now you run this add repo code explicitly only on SLE.